src/components/Contentful/contentful_fonts.scss

```diff
@@ -0,0 +1,138 @@
+.new-design {
+  $mt-green-dark: #31aa33;
```

Comment:
You may need to move these out into a settings directory once they are required in multiple files similar to the madetech/frontend module: https://github.com/madetech/frontend/blob/master/src/settings/_colors.scss

Reply:
Yep, will keep that in mind.